ext/fs/ops.rs

```diff
@@ -1362,20 +1382,21 @@ fn op_fs_fstat_sync(
   Ok(())
 }
 
-#[op]
-async fn op_fs_fstat_async(
+#[op2(async)]
+#[serde]
+pub async fn op_fs_fstat_async(
   state: Rc<RefCell<OpState>>,
-  rid: ResourceId,
+  #[smi] rid: ResourceId,
 ) -> Result<SerializableStat, AnyError> {
   let file = FileResource::get_file(&state.borrow(), rid)?;
   let stat = file.stat_async().await?;
   Ok(stat.into())
 }
 
-#[op]
-fn op_fs_flock_sync(
+#[op2(fast)]
+pub fn op_fs_flock_sync(
```

Comment:
**question:** Does it make sense that the op functions now need to be public? It seems to make sense, since eg. in this file it seems there isn't anything else public except the op functions, so someone somewhere must be importing those and "injecting them into the runtime" so to say.

I guess another way to ask this question is: Did the old op macro already always create public functions, and does op2 change this to eg. only generate public functions if the original function was public or something like that?

Reply:
The old op would generate `pub` op structs regardless, so this is "more correct". In theory they don't need to be `pub`, but in this case they'd have to at least be `pub(crate)`.

Reply:
In that case I am very happy!